db/queries/assessment_records/_helpers.py

```diff
@@ -61,5 +61,17 @@ def derive_application_values(application_json):
         derived_values["location_json_blob"] = application_json[
             "location_json_blob"
         ]
+    else:
+        derived_values["location_json_blob"] = {"error": True}
+
+    FIELD_DEFAULT_VALUE = "Not Available"
+    if derived_values["location_json_blob"]["error"] is True:
+        derived_values["location_json_blob"]["county"] = FIELD_DEFAULT_VALUE
+        derived_values["location_json_blob"]["region"] = FIELD_DEFAULT_VALUE
+        derived_values["location_json_blob"]["country"] = FIELD_DEFAULT_VALUE
```

Comment:
I would prefer leaving these as `None` as this label is presentational and should be controlled in the frontend rather than set in the database.